README.Rmd

```diff
@@ -0,0 +1,56 @@
+
+---
+title: "kbtbr"
+output: github_document
```

Comment:
I found it always bothersome that, on manual rendering, this might also create the HTML version.
It can be disabled like:
```
output:
    github_document:
        html_preview: false
```

Reply:
fixed